### ISSUE TITLE
Points don't render when both href and tooltip aesthetics are used

### ISSUE THREAD

**Issue by analyticalmonk** [tdhock/animint#155]
For example,
```
library(animint)
data(WorldBank, package = "animint")

ex_plot <- ggplot() +
    geom_point(aes(fertility.rate, life.expectancy, color = region,
                   tooltip = country, href = "https://github.com"),
               data = WorldBank[WorldBank$year == 1975, ])

plot_list <- list(ex = ex_plot)
structure(animint, class = "animint")
```
The above code results in an animint plot without the required points.
![animint_hreftooltipissue2](https://cloud.githubusercontent.com/assets/4419448/16407237/50bfeafa-3d30-11e6-9f23-f915d6ffb16c.png)

Removing either of the two (tooltip or href) provides the expected plot.
![animint_tooltiphrefissue](https://cloud.githubusercontent.com/assets/4419448/16407259/6624dbb2-3d30-11e6-8c22-ab79ffe22ec8.png)

@faizan-khan-iit Please look into this.

_Note: As discussed in animint issue #100 and Rperform issue [#15](https://github.com/analyticalmonk/Rperform/issues/15)_

**Comment by faizan-khan-iit** [comment]
@analyticalmonk Thanks for pointing this out Akash. I will take a look into this ASAP.

**Comment by faizan-khan-iit** [comment]
@analyticalmonk Hey Akash, the problem with using both `href` and `tooltip` has been fixed.

Here's your example - http://bl.ocks.org/faizan-khan-iit/raw/d8671bf1b2450213348b39373d7a31e1/

Please inform if you have any problem working with this.
Cheers!

**Comment by faizan-khan-iit** [comment]
Closed by #156
